### PR TITLE
Display program deep link to admin

### LINKED ISSUE #1298: Add admin UI that allows for copying of deep link to a program.
**Is your feature request related to a problem? Please describe.**
Applicant flow into CiviForm will require applicants to enter through an "apply now" link on program landing pages maintained by the city.  Links to start a program-specific application need to be available for end-to-end applicant flow.  

**Describe the solution you'd like**
Allow a sharable / copyable link to program specific applications.  And ask language question when starting application after clicking link.

**Additional context**

Flow:  
image

Language question (currently only asked in main civiform entry point):  
image

**Done when**
When program specific link can be accessed and copied by the CiviForm Admin.

### PR DESCRIPTION
**Pull request by bion**
Screen Shot 2021-06-02 at 4 17 40 PM

fixes #1298

### CODE REVIEW

File: universal-application-tool-0.0.1/app/views/admin/programs/ProgramIndexView.java [hunk 1]
```diff
@@ -124,6 +128,16 @@ public Tag renderProgramListItem(
         div(programDescriptionText)
             .withClasses(Styles.TEXT_GRAY_700, Styles.TEXT_BASE, Styles.MB_8, Styles.LINE_CLAMP_3);
 
+    Tag programDeepLink =
+        input()
```
Comment (natsid) [inline universal-application-tool-0.0.1/app/views/admin/programs/ProgramIndexView.java:132]: Why use an input for this? Is it better for copying?

**Review by natsid — APPROVED** [review]
Nice - thanks for adding this quickly!

File: universal-application-tool-0.0.1/app/views/admin/programs/ProgramIndexView.java [hunk 2]
```diff
@@ -124,6 +128,16 @@ public Tag renderProgramListItem(
         div(programDescriptionText)
             .withClasses(Styles.TEXT_GRAY_700, Styles.TEXT_BASE, Styles.MB_8, Styles.LINE_CLAMP_3);
 
+    Tag programDeepLink =
```
Comment (natsid) [inline universal-application-tool-0.0.1/app/views/admin/programs/ProgramIndexView.java:131]: Suggestion (non-blocking): Maybe add a short string explaining what this URL is.

File: universal-application-tool-0.0.1/app/views/admin/programs/ProgramIndexView.java (hunk 1 above)
Comment (bion) [reply universal-application-tool-0.0.1/app/views/admin/programs/ProgramIndexView.java:132]: Yeah, just a bigger click target for double-clicking to highlight

**Review by natsid — APPROVED** [review]
LGTM

File: universal-application-tool-0.0.1/app/views/admin/programs/ProgramIndexView.java (hunk 2 above)
Comment (bion) [reply universal-application-tool-0.0.1/app/views/admin/programs/ProgramIndexView.java:132]: Done

**Pull request merged**